sdk/eventhub/azure-eventhub-checkpointstoreblob-aio/azure/eventhub/extensions/checkpointstoreblobaio/_vendor/storage/blob/_shared/avro/__init__.py

```diff
@@ -1,13 +1,5 @@
-# coding=utf-8
-# --------------------------------------------------------------------------
```

Comment:
This is a bit weird - but I think it's just a Github diff issues...

Reply:
the latest storage blob doesn't have this file any more, let's pretend it doesn't exist 😄